index.html

```diff
@@ -114,11 +114,12 @@
         </li>
       </ul>
       <p>
-        The goal of this specification is to define the behaviors of Fedora server implementations in order to
-        facilitate interoperability with client applications.  Although Fedora servers are not necessarily drop-in replacements
-        for one another, a client developed against this specification should work against any Fedora server with little modification.
-        Some implementations may be unable to provide all features described in this document due to conscious design decisions, but
-        at least the method of rejection for requests against such features is defined.
+        The goal of this specification is to define the fundamental behaviors and interaction
+        patterns of a Fedora server. A consequence of this goal is that client appications will
+        have a well-defined model for interacting with different Fedora implementations.
```

Comment:
I think you are right, @escowles, that this is the heart of the disagreement. With that in mind, would it not be appropriate to remove discussion of goals from the spec (goals that, it seems clear, are not held in common) and confine the introductory material to discussion of what concrete affordances the spec will provide (i.e. `the interaction patterns defined in this specification should provide...`)?